src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToStringTransformer.php

```diff
@@ -94,24 +94,25 @@ public function __construct($inputTimezone = null, $outputTimezone = null, $form
      *                                       instance or if the output timezone
      *                                       is not supported.
      */
-    public function transform($value)
+    public function transform($dateTime)
```

Comment:
3) for this one :)
4) usually: PR against 2.3, then we merge up to master (opposite direction from 2))